docs/overview/README.md

```diff
@@ -1,18 +1,27 @@
 # Overview
 
-The IOTA client library is a stateless library that provides a high level abstractions on top of IOTA node software to help interact with IOTA network in user friendly way and allows you to do the following, for instance:
+To communicate with the IOTA network, you have to connect and interact with a [node](https://chrysalis.docs.iota.org/node-software/node-software.html). This library, the IOTA Client, wraps routine tasks and minute details into its tidy API. It will generate addresses for you, send messages, sign and send transactions, and more.
```

Comment:
Just for consistency's sake, are we capitalizing "Client", fullstop or only in some cases?